packages/loader/src/Loader.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Ripple } from '@onaio/loaders';
+
+/** interface for Ripple props */
+export interface LoaderProps {
```

Comment:
@p-netm @KipSigei @dubdabasoduba Should we really have this component configurable? I was thinking the `opensrp/loader` should not allow configuration so that the loader can be standard throughout the OpenERP package?

Reply:
i do agree with you, we shouldn't make this component configurable

Reply:
@kelvin-muchiri @ashfahan what's stopping us from using it in standardized way if its configurable.

Reply:
@p-netm Say I import the package in user-management, I should not be allowed to pass my own configuration which will be different from location-management. i.e The loader should be exactly the same across the across components, across other packages

Reply:
In embodiment of the principles that we wish the codebase to have. I do not think we can justify saying that any piece of code shouldn't be configurable, especially given that other devs are the intended consumers of the piece of code in question. In my view in such a case we should set the defaults to reflect what we want the standard to be, but still keep things flexible.
however this is just my personal preference.

Reply:
@p-netm  if we really need a configurable packages why dont we direactly the loader packages from jstools? .. i mean it does have the default one set to ripple right ? we really dont need have an extra wrapper around jstools's one ...

whats the points of having this extra loader packages ? i thought we are using this one to have our own *static* configured one *just for this project only*

Reply:
@p-netm I also thought at first that we should leave it configurable. But again like in my earlier example, it also means rationally, my code should pass code review if I pass my own configuration when I use the loader in a component. i.e It would be valid to pass my own config different from other components